gitlab.yml

```diff
@@ -41,21 +41,21 @@ functions:
       - gateway_config.yml
     secrets:
       - gitlab-api-token
-      - payload-secret 
-      
+      - payload-secret
+
   gitlab-push:
     lang: go
     handler: ./gitlab-push
-    image: functions/gitlab-push:0.2.1
+    image: functions/gitlab-push:0.3.0
     labels:
       openfaas-cloud: "1"
       role: openfaas-system
       com.openfaas.scale.zero: false
     environment:
       write_debug: true
       read_debug: true
+      branches: "master,"
```

Comment:
Can this still work even as `branches: "master"` without the `"master,`" (comma)?

Reply:
Yes I have added case in the unit tests but I will remove this functionality

Reply:
Now this is removed 👍